commerce/domain/shipping/splitshipment/SplitShipmentUnitTest.cls

```diff
@@ -0,0 +1,303 @@
+/**
+ * @description Sample unit test for SplitShipmentSample and SplitShipmentCallsSuper.
+ */
+@IsTest
+global class SplitShipmentUnitTest {
+
+    private static final MockDefaultSplitShipmentService mockService = new MockDefaultSplitShipmentService();
+    private static final SplitShipmentCallsSuper splitShipmentCallsSuper = new SplitShipmentCallsSuper(mockService);
+    private static final SplitShipmentSample splitShipmentSample = new SplitShipmentSample();
+
+    @IsTest
+    public static void splitShipmentApexThatCallsSuperShouldInvokeDefaultSplitShipmentService() {
+        // Arrange
+        CartExtension.Cart cart = CartExtension.CartTestUtil.createCart();
+
+        CartExtension.ItemArrange itemArrange = new CartExtension.ItemArrange.Builder()
+                .withProductId('01txx0000006iP2AAI')
```

Comment:
Why ItemArrangement has both ProductId and CartItemId? What if ProductId supplied is different from CartItem.ProductId?